pyproject.toml

```diff
@@ -26,13 +26,13 @@ classifiers = [
     "Topic :: Scientific/Engineering",
     "Development Status :: 3 - Alpha"
 ]
-requires-python = ">=3.8,<3.11"
+requires-python = ">=3.9, <3.10"
```

Comment:
This indeed is a quirk in `requires-python`. It doesn't accept `==` so we must use the restricted range. Really weird...
something to keep in mind.